demos/browser/app/meetingV2/meetingV2.html

```diff
@@ -45,10 +45,11 @@ <h1 class="h3 mb-3 font-weight-normal">Join a meeting</h1>
       <div class="row mt-3">
         <div class="col-12">
           <label for="optional-features">Optional Features</label>
-          <select id="optional-features" class="custom-select" style="width:100%" multiple>
```

Comment:
Does that mean I cannot enable simulcast and unified plan option at the same time?

Reply:
That's right, it can not be enabled at the same time. The reason I added this is to allow easy testing for QA and the multiple selection confuses them.